content/SCALE/SCALETutorials/Apps/CommunityApps/MinIOApp/_index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "MinIO Apps"
```

Comment:
I think you should do this (or something similar). It would fix both the problem with header levels in this article and make navigation less confusing. In the children list right now it says:

Related Content
-Installing MinIO Enterprise App
-Setting Up MinIO Clustering
-Preparing for Clustering
-MinIO Apps

I would not know to click on MinIO Apps to get here.


```suggestion
title: "Installing MinIO (S3) Community App"
```

Reply:
Per our discussion, I'll add the missing ToC and the new H2 to introduce the detailed explanation sections.